src/main/java/WebDefinitionSteps.java

```diff
@@ -0,0 +1,26 @@
+import common.AbstractDefinitionSteps;
+import net.thucydides.core.annotations.Steps;
+import org.jbehave.core.annotations.Given;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.annotations.When;
+
+public class WebDefinitionSteps extends AbstractDefinitionSteps {
+
+    @Steps
+    private WebSteps webSteps;
+
+    @Given("User open '$page' page")
+    public void userOpenPage(final String page) {
+        webSteps.openPageByParameter(page);
+    }
+
+    @When("User search '$searchedText' to '$elementName' on '$page' page")
```

Comment:
Using steps like this you need to control Page Name all the time, I'm not sure it's a good approach. Let's discuss.